tests/tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = {ocs-tests}
+basepython = py37
+skipsdist = true
+
+[testenv]
+deps=
+  -rrequirements.txt
```

Comment:
is that typo ( -r requirements.txt )

Reply:
it's not typo @vavuthu , if you will do pip install -rrequirements.txt it work like a char as just -r is the first argument and then the path to requirement file following. It's more readable when there is space I agree, but doesn't matter.

Reply:
Do we really need to separate and have other tox.ini file in tests directory? Cannot we just have different env defined in tox.ini file in root directory of the project?

Reply:
We could have a different env defined in the tox.ini file in the root. With as simple as it is now, that might be a good option. I've found in the past though that multiple, less complicated tox.ini files are better than trying to manage one very complicated tox.ini that does everything.